docs/examples/index.rst

```diff
@@ -93,10 +96,10 @@ you might find interesting.
     customizing_visualization
     move_schemes_strategies
 
-In addition, there are a set of examples that are hosted outside the core
-OPS repository. These are the OPS additional examples, primarily developed
-as part of E-CAM, a European Union Horizon 2020 project. Those examples are
-available at https://gitlab.e-cam2020.eu/dwhswenson/ops_additional_examples.
+More examples can be found in the `OPS additional examples repository
+<http://github.com/openpathsampling/ops_additional_examples>`_. Live
+versions of those examples are also available through `binder
+<https://mybinder.org/v2/gh/openpathsampling/ops_additional_examples/HEAD>`_.
```

Comment:
Huh, I wanted to propose to let this one spin up into jupyter-lab as well, but adding `?urlpath=lab` makes the binder `404`. Still, making this work in jupyter-lab via binder might be nice for consistency on the page (as the other link does go to jupyter-lab)

Reply:
Couldn't you technically run them while building the binder? (by running: `jupyter nbconvert --to notebook --execute mynotebook.ipynb`